vdr/didx509/jwt_utils.go

```diff
@@ -0,0 +1,33 @@
+package didx509
+
+import (
+	"context"
+	"errors"
+	"github.com/lestrrat-go/jwx/v2/jws"
+)
+
+var (
+	ErrorInvalidNumberOfSignatures = errors.New("invalid number of signatures")
+)
+
+// ExtractProtectedHeaders extracts the protected headers from a JWT string.
+// The function takes a JWT string as input and returns a map of the protected headers.
+// Note that:
+//   - This method ignores any parsing errors and returns an empty map instead of an error.
+func ExtractProtectedHeaders(jwt string) (map[string]interface{}, error) {
```

Comment:
should be private otherwise it's in the wrong place

Reply:
It's not private, so where should it go? `crypto/jwx.go`?

Reply:
This method is made private.

Reply:
Done.